data_validation/query_builder/query_builder.py

```diff
@@ -466,15 +466,13 @@ def compile_calculated_fields(self, table, n=0):
         # else:
         #     return [field.compile(table) for field in self.calculated_fields]
 
-    def compile(self, validation_type, data_client, schema_name, table_name):
+    def compile(self, validation_type, table):
         """Return an Ibis query object
 
         Args:
-            data_client (IbisClient): The client used to validate the query.
-            schema_name (String): The name of the schema for the given table.
-            table_name (String): The name of the table to query.
+            table (IbisTable): The Ibis Table expression.
         """
-        calc_table = clients.get_ibis_table(data_client, schema_name, table_name)
+        calc_table = table
```

Comment:
Instead of renaming this, let's replace the instances of 'calc_table' with 'table'.

Reply:
Fixed through this [commit](https://github.com/GoogleCloudPlatform/professional-services-data-validator/pull/607/commits/0e6ac6965696b40cbcf577c17983bee8ae24c92a)